src/main/java/pro/trevor/tankgame/ui/rpc/Cli.java

```diff
@@ -23,7 +23,7 @@ public Cli(RpcHandler handler) {
             RpcMethod rpcMethod = method.getAnnotation(RpcMethod.class);
             if(rpcMethod == null) continue;
 
-            if(!method.getReturnType().equals(JSONObject.class)) {
+            if(!method.getReturnType().isInstance(JSONObject.class)) {
```

Comment:
I thought isInstance checked if an Object is an instance of that class. This would always evaluate to false.